File: src/jef/compiler/template/text_node.rb
# To change this template, choose Tools | Templates
# and open the template in the editor.

require 'template/template_node'

class TextNode < TemplateNode
  
	@expression
	
	def initialize str
    super nil
		@expression = TextExpression.new str
  end
	
	def get_js_expression
		return @expression.js_expression
	end
	
end

File: src/tomek.js/compiler.js/template/TextNode.ts
import TemplateNode from "./TemplateNode";
import TextExpression from "./TextExpression";

export default class TextNode extends TemplateNode {

	raw: string;
	expression: TextExpression;

	constructor(text: string) {
		super();
		this.raw = text;
		this.expression = new TextExpression(text);
	}

	getJsExpression(){
		return this.expression.epression;
	}

	detailedDescription(){
		return this.expression.epression;
	}

}